rules.d/99-systemd.rules.in

```diff
@@ -15,6 +15,11 @@ KERNEL=="vport*", TAG+="systemd"
 SUBSYSTEM=="ubi", TAG+="systemd"
 
 SUBSYSTEM=="block", TAG+="systemd"
+
+# Suspended dm-crypt devices will be marked as plugged if they were marked as ready previously.
+SUBSYSTEM=="block", ENV{DM_UUID}=="CRYPT-*", ENV{ID_PART_TABLE_TYPE}=="", ENV{ID_FS_USAGE}=="", ENV{DM_SUSPENDED}=="1", IMPORT{db}="SYSTEMD_READY"
```

Comment:
why the conditionalization on ID_FS_USAGE/ID_PART_TABLE_TYPE? why not drop that? can you explain?